prov/rxm/src/rxm.h

```diff
@@ -646,6 +646,18 @@ struct rxm_msg_eq_entry {
 #define RXM_CM_ENTRY_SZ (sizeof(struct fi_eq_cm_entry) + \
 			 sizeof(union rxm_cm_data))
 
+typedef ssize_t (*rx_buf_handle_fn_t)(struct rxm_rx_buf *rx_buf);
+
+typedef int (*tx_handle_eager_fn_t)(struct rxm_ep *rxm_ep,
+				    struct rxm_tx_eager_buf *tx_eager_buf);
+
+struct rxm_handle_txrx_ops {
+	tx_handle_eager_fn_t	tx_eager;
```

Comment:
tx_eager implies that it is transmitting an eager message, but it's set to function named finish_eager_send.  Please rename to clarify its use.  Is this handling a tx completion?  comp_eager_tx?